### ISSUE TITLE
[Enhancement] Sort order of albums: Ignore images without timestamps

### ISSUE THREAD

**Issue by vorlif** [LycheeOrg/Lychee#738]
Hello,

I sort my albums by `max_takestamp DESC`. Unfortunately I have a few albums with pictures that have no timestamp. These albums are always displayed at the first position. It would be nice if the images without timestamps were ignored in the sorting order.

**Comment by ildyria** [comment]
?

**Comment by kamil4** [comment]
I was going to say that while ignoring the images without timestamps would be hard, we do plan to add a capability to set/edit the timestamp; that way you could always ensure that every image has a timestamp of your choice...

**Comment by vorlif** [comment]
Sorry, that was my mistake. I called `php artisan lychee:takedate` after I set the timestamp manually with exiftool. After that I was surprised that the sorting order did not change. I thought it was because of the images without timestamps. But the problem is that `lychee:takedate` does not update the fields `min_takestamp` and `max_takestamp` of table `albums`.

I have performed the update manually. 

Thanks for your time and effort!
